app/presenters/cost_summary/work_items.rb

```diff
@@ -15,6 +15,7 @@ def rows
       work_types.map do |work_type|
         total_cost = forms[work_type]&.sum(&:total_cost)
         {
+          head_key: work_type.to_s,
```

Comment:
not a fan of adding this to the hash to make it work in two spots... any reason we can use the translated text???

i.e.
```

    def row_data
      header_rows + work_items.rows.map do |row|
        {
          head_key: row.fetch(:key, :text),
          text: row[:value][:text]
        }
      end
    end
```

Reply:
rows works in such a way that head_key is taken from row_data for each element and the translated value is fetched using the key (see CheckAnswers::Base::row_content)

If I change row_content to  just take a text value instead of taking a key, I could do this. It would make the rest of the code a lot more verbose however